open_vsdcli/vsd_gateway.py

```diff
@@ -204,3 +204,45 @@ def bridgeinterface_list(ctx, filter, **ids):
                        line['name'],
                        line['VPortID']])
     print table
+
+
+@vsdcli.command(name='gatewayredundancygroup-list')
+@click.option('--enterprise-id', metavar='<ID>')
+@click.pass_context
+def gatewayredundancygroup_list(ctx, enterprise_id):
+    """list all gateway redundant groups"""
+    if enterprise_id:
+        url_request = "enterprises/%s/redundancygroups" % enterprise_id
+    else:
+        url_request = "redundancygroups"
+    result = ctx.obj['nc'].get(url_request)
+    table = PrettyTable(["ID",
+                         "Redundant Gateway Status",
+                         "Name",
+                         "Description",
+                         "Personality",
+                         "externalID"])
+    for line in result:
+        table.add_row([line['ID'],
+                       line['redundantGatewayStatus'],
+                       line['name'],
+                       line['description'],
+                       line['personality'],
+                       line['externalID']])
```

Comment:
externalID removed
Choose to use gatewaypeerXName instead of gatewaypeerXID to limit line size in output
gatewaypeerXID is available in show option